class DomainError(Exception):
    pass


class NotFoundError(DomainError):
    pass
